projects/plugins/jetpack/tests/e2e/specs/post-connection/search-dashboard.test.js

```diff
@@ -0,0 +1,67 @@
+import { test, expect } from '../../fixtures/base-test.js';
+import { SearchDashboard } from 'jetpack-e2e-commons/pages/wp-admin/index.js';
+import { enableInstantSearch, disableInstantSearch } from '../../helpers/search-helper.js';
+import { prerequisitesBuilder, Plans } from 'jetpack-e2e-commons/env/index.js';
+import playwrightConfig from '../../playwright.config.cjs';
+
+test.describe( 'Search Dashboard', () => {
+	let searchDashboard;
+
+	test.beforeAll( async ( { browser } ) => {
+		const page = await browser.newPage( playwrightConfig.use );
+		await prerequisitesBuilder( page )
+			.withLoggedIn( true )
+			.withConnection( true )
+			.withPlan( Plans.Complete )
+			.withActiveModules( [ 'search' ] )
+			.build();
+
+		await enableInstantSearch();
+		await page.close();
+	} );
+
+	test.afterAll( async () => {
+		await disableInstantSearch();
+	} );
+
+	test.beforeEach( async ( { page } ) => {
+		searchDashboard = await SearchDashboard.visit( page );
+		await searchDashboard.waitForNetworkIdle();
```

Comment:
I wonder why waiting for the network here and not leaving the default `domcontentloaded` included in the visit function.
Is there a specific search request needed?
I'm also asking because I know this is a fragile point in the other existing search tests also, making them fail quite often.

Reply:
Thanks for the suggestion, addressed in c477b27ebf.
I assumed the visit function would wait for the page to be ready but apparently it doesn't.